packages/shared/src/components/sidebar/SquadsList.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactElement } from 'react';
+import { Squad } from '../../graphql/squads';
+import DefaultSquadIcon from '../icons/DefaultSquad';
+import NewSquadIcon from '../icons/NewSquad';
+import { ClickableNavItem } from './ClickableNavItem';
+import { ItemInner, NavItem, SidebarMenuItem } from './common';
+import { Image } from '../image/Image';
+import { cloudinary } from '../../lib/image';
+
+type SquadsListProps = {
+  squads: Squad[];
+  onNavTabClick?: (page: string) => unknown;
+};
+
+const SquadImage = ({ image, name }: Squad) => (
+  <Image
+    title={name}
+    src={image}
+    fallbackSrc={cloudinary.squads.imageFallback}
+    className="object-cover w-5 h-5 rounded-full"
+    loading="lazy"
+  />
+);
+
+export function SquadsList({
+  squads,
+  onNavTabClick,
+}: SquadsListProps): ReactElement {
+  const newSquadMenuItem: SidebarMenuItem = {
+    icon: () => <NewSquadIcon />,
+    title: 'New squad',
+    path: '/new-squad',
+  };
+
+  return (
+    <>
+      {squads.map((squad) => {
+        const { handle, name, permalink, image } = squad;
+        const menuItem: SidebarMenuItem = {
+          icon: () =>
+            image ? <SquadImage {...squad} /> : <DefaultSquadIcon />,
+          title: name,
+          path: permalink,
+          action: () => onNavTabClick?.(`squads/${handle}`),
```

Comment:
Do you know what this callback (`onNavTabClick`) does?
Because I'm not sure what's the right parameter to pass here

Reply:
I think it changes the page to webapp if it is clicked in the extension

Reply:
In this case, we should use `permalink` don't we?
The backend already returns the exact link we should use

Reply:
onNavTabClick is used with relative URLs everywhere else. That's why I created a relative URI instead of using the permalink which is absolute
<img width="384" alt="image" src="https://user-images.githubusercontent.com/1488164/209785803-5b8c0c9d-a74c-4edb-b02f-d583c53f4331.png">

Reply:
Ok, let's keep it for the final review, then. I'd love Chris to share his thought. I'm not 100% sure what's going on behind the scenes

Reply:
I think onNavClick is only for items that are in the extension.
This always goes to webapp so think indeed permalink should be used.

You can double check how we link reading history :)

Reply:
@rebelchris thanks for jumping in! 🙏

Reply:
Ok, removed onnavclick 👍 (we already have path)